src/figvalidationservice/pom.xml

```diff
@@ -134,6 +134,14 @@
             <version>2.2</version>
         </dependency>
         
+        <!-- Used for testing -->
+		<dependency>
+		    <groupId>junit</groupId>
+		    <artifactId>junit</artifactId>
+		    <version>4.13</version>
```

Comment:
I have done the same thing on dps validation service, but noticed that our test are using junit 5, I will check if we can remove this dependency, mostly using for assert...